src/test/objectstore/store_test.cc

```diff
@@ -55,6 +55,20 @@ typedef boost::mt11213b gen_type;
 const uint64_t DEF_STORE_TEST_BLOCKDEV_SIZE = 10240000000;
 #define dout_context g_ceph_context
 
+static uint64_t get_testing_seed(const char* function) {
+  char* random_seed = getenv("TEST_RANDOM_SEED");
```

Comment:
How about simpler env name?
My proposal is `SEED`.

Reply:
I didn't want to create a generic name that might conflict with other programs tbh